app/main.py

```diff
@@ -1,3 +1,11 @@
-def shop_trip():
-    # write your code here
-    pass
+from app.trip.trip import Trip
+from app.customer.get_customers import get_list_of_customers
+from app.shop.get_shops import get_list_of_shops
+
+
+def shop_trip() -> None:
+    shops = get_list_of_shops()
+    customers = get_list_of_customers()
+
+    for customer in customers:
+        Trip(customer, shops)
```

Comment:
You can initialize Trip class once above the loop

Then you can create go_shopping method and call it for every customer